contrib/setup-bgp-peer.sh

```
@@ -0,0 +1,251 @@
+  #                                                                            ---------------------
+  local ipv6_network
+  if [ "$PLATFORM_IPV4_SUPPORT" == true ] && [ "$PLATFORM_IPV6_SUPPORT" == true ]; then
+    ipv6_network="--ipv6 --subnet=${BGP_SERVER_NET_SUBNET_IPV6}"
+  elif  [ "$PLATFORM_IPV6_SUPPORT" == true ]; then
+    ipv6_network="--ipv6 --subnet=${BGP_SERVER_NET_SUBNET_IPV6}"
+  else
+    ipv6_network=""
+  fi
+  $OCI_BIN rm -f bgpserver
+  $OCI_BIN network rm -f bgpnet
+  $OCI_BIN network create --subnet="${BGP_SERVER_NET_SUBNET_IPV4}" ${ipv6_network} --driver bridge bgpnet
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)